src/components/Content/Content.styled.tsx

```diff
@@ -0,0 +1,107 @@
+import styled from "styled-components";
+import theme from "../../theme";
+import EditIcon from "@mui/icons-material/Edit";
+
+export const MainWrapper = styled.div`
+	font-family: ${theme.fontFamily};
+	border-bottom: 2px solid ${theme.colors.gray200};
+`;
+
+export const TitleWrapper = styled.div`
+	align-items: center;
+	display: flex;
+	justify-content: space-between;
+`;
+
+export const Title = styled.span`
+	color: ${theme.colors.almostBlack};
+	font-size: 16px;
+	font-weight: ${theme.fontWeight.semiBold};
+`;
+
+export const StyledEditIcon = styled(EditIcon)`
+	color: ${theme.colors.gray600};
+	cursor: pointer;
+
+	&.MuiSvgIcon-root {
+		border-right: 1px solid ${theme.colors.gray200};
+		padding-right: 16px;
+		width: 16px;
+	}
+`;
+
+export const ButtonsWrapper = styled.div`
+	align-items: center;
+	display: flex;	
+
+	.MuiButton-root {
+		padding-left: 16px;
+	}
+
+	.iconButton {
+		border-right: 1px solid ${theme.colors.gray200};
+		padding-right: 16px;
+	}
+`;
+
+// Styles for the paragraph content type
+export const Paragraph = styled.p`
+	color: ${theme.colors.gray700};
+	font-size: 14px;
+	margin: 0;
+`;
+
+export const ParagraphWrapper = styled.div`
+	margin-bottom: 16px;
+`;
+
+// Styles for the LabelValue content type
+
+export const LabelValueWrapper = styled.div`
```

Comment:
Isn't this the same as the paragraph wrapper?

Reply:
You are right, I will use the same wrapper for both.